signac/__main__.py

```diff
@@ -32,7 +32,7 @@
 from . import get_project, init_project
 from .common import config
 from .common.configobj import Section, flatten_errors
-from .contrib.filterparse import _add_prefix, parse_filter_arg
```

Comment:
My guess is that `_add_prefix` can be removed entirely at this point. I can verify that and create a separate issue for that though.

Reply:
#810